scripts/install

```diff
@@ -248,6 +248,12 @@ sed 's|<TPL:NGINX_REALPATH>|/var/www/wifiadmin/|g' -i /etc/php5/fpm/pool.d/wifia
 # Fix sources
 sed "s|<TPL:NGINX_LOCATION>|${path_url}|g" -i /var/www/wifiadmin/config.php
 
+# Fix network interface
+grep -q -x -F 'auto wlan0' /etc/network/interfaces || echo 'auto wlan0' >> /etc/network/interfaces
```

Comment:
(Can we also grep in `interfaces.d` just in case ? :wink: )

Reply:
ok I'll do ! Thanks